utils/env_vars.go

```diff
@@ -7,6 +7,18 @@ import (
 	"strconv"
 )
 
+func GetRequiredIntEnv(envVarName string) int {
+	envValue, ok := os.LookupEnv(envVarName)
+	if !ok || envValue == "" {
+		log.Fatalf("%s environment variable is required", envVarName)
+	}
+	intValue, err := strconv.Atoi(envValue)
```

Comment:
for what it's worth, technically this by itself covers all the errors you can have above too

Reply:
Mmh... I think I miss something. How do you use `strconv.Atoi` without calling `os.LookupEnv` first here ?

Reply:
Hmm non en fait ce que je voulais dire (mais c'était peut-être con) c'était que tu pourrais faire un LookupEnv sans même check le "ok" et passer directement l'output (au pire c'est une string vide) à Atoi() puis check l'erreur juste là
mais c'était pas forcément un commentaire hper pertinent